drivers/unix/net_socket_posix.cpp

```diff
@@ -187,24 +187,31 @@ NetSocketPosix::NetError NetSocketPosix::_get_socket_error() const {
 #if defined(WINDOWS_ENABLED)
 	int err = WSAGetLastError();
 
+	// More into here https://docs.microsoft.com/en-us/windows/win32/winsock/windows-sockets-error-codes-2
 	if (err == WSAEISCONN)
 		return ERR_NET_IS_CONNECTED;
 	if (err == WSAEINPROGRESS || err == WSAEALREADY)
 		return ERR_NET_IN_PROGRESS;
 	if (err == WSAEWOULDBLOCK)
 		return ERR_NET_WOULD_BLOCK;
+	if (err == WSAEADDRINUSE || err == WSAEADDRNOTAVAIL)
+		return ERR_NET_ADDRESS_INVALID_OR_UNAVAILABLE;
+	if (err == WSAEACCES)
+		return ERR_NET_UNAUTHORIZED;
 	print_verbose("Socket error: " + itos(err));
 	return ERR_NET_OTHER;
 #else
-	if (errno == EISCONN) {
+	// More info here https://man7.org/linux/man-pages/man2/bind.2.html
+	if (errno == EISCONN)
```

Comment:
I don't know that this is allowed by the Godot style guidelines. @NathanLovato opinions?

Either way, Im not a fan of changing existing code style willy nilly. It convolutes the PR unnecessarily.

Reply:
That's a wrong change indeed, braces are required.

Reply:
My personal preference is to always use braces on `if` blocks.
In this case, I was just trying to follow the same code style that was used in the lines above.
I'll revert this change.